src/pages/Components/ComponentsInfo/LinksInfo.jsx

```diff
@@ -131,6 +131,26 @@ const linkListHtml = `
 </a>
 `;
 
+const tableLink = `
+<TableLink title="Tabell 1" href="" hrefText="Gjennomsnittspris i fritt salg på eid tomt i 100-meterbeltet. 2015-2108." />
+`;
+
+const tableLinkHtml = `
+<div className="ssb-table-link">
+    <a href="">
+        <div className="tl-icon">
+            <i>{feather.arrowRight 22px}</i>
```

Comment:
Just a minor detail for consistencies sake; should the text inside the i-tags also be "{insert icon here}", as with the other links with icons?